.github/workflows/cleanup-images.yml

```
@@ -0,0 +1,44 @@
+    strategy:
+      matrix:
+        package: [rate-ukma/backend, rate-ukma/webapp]
+    steps:
+      - name: Delete tagged versions older than 30 days
+        uses: actions/delete-package-versions@v5
+        with:
+          package-name: ${{ matrix.package }}
+          package-type: container
+          min-versions-to-keep: 5
+          delete-only-untagged-versions: false
+          older-than: 14
```

actionlint (1.7.12)

[error] input "older-than" is not defined in action "actions/delete-package-versions@v5". available inputs are "delete-only-pre-release-versions", "delete-only-untagged-versions", "ignore-versions", "min-versions-to-keep", "num-old-versions-to-delete", "owner", "package-name", "package-type", "package-version-ids", "token"

(action)